stardis/opacities/base.py

```diff
@@ -20,41 +20,27 @@ def read_wbr_cross_section(wbr_fpath):
     return wbr_cross_section
 
 
-def calc_hminus_density(h_neutral_density, temperature, electron_density):
-    thermal_de_broglie = ((THERMAL_DE_BROGLIE_CONST / temperature) ** (3 / 2)).to(
-        u.cm**3
-    )
-    phi = (thermal_de_broglie / 4) * np.exp(H_MINUS_CHI / (const.k_B * temperature))
-    return h_neutral_density * electron_density * phi.value
-
-
 def calc_tau_h_minus(
     splasma,
```

Comment:
just rename to `splasma` to `plasma`.

Reply:
That would have to be a separate PR, it is ``splasma`` everywhere in the code -- I personally like it but I can make a PR changing every ``splasma`` to ``plasma``.

If I did it here, it may mess with the PRs based on this branch.